futaba/cogs/moderation/core.py

```diff
@@ -121,6 +121,13 @@ async def perform_mute(
         if roles.mute_role in member.roles:
             raise CommandFailed(content="User is already muted")
 
+        # Check that users top role is not the same as the requesters top role.
+        if member != ctx.author:
+            if member.top_role == ctx.author.top_role:
+                raise CommandFailed(
+                    content="You can not mute a user with the same role as you"
```

Comment:
"same role" is confusing, consider something like "You can only mute users lower than you"

Reply:
@JoshuaS3